src/data.js

```diff
@@ -824,16 +824,15 @@ export const vitals = [
     id: "Humanity",
     style: "humanityPt",
     scale: 10,
-    value: null,
-    defaultValue: 7, 
```

Comment:
som to premenoval na modifier, lebo su to 2 ine veci podla mna... toto si zkazil
modifier ze to nejak upravuje tie hodnoty na ktorych je to zavisle, default value ked neexistuje value

Reply:
sorry, zdalo sa mi, ze ostatne staty nemaju value, tak som to zmazal. Vlastne neviem preco to tam je ten null a zdalo sa mi, ze vsetko funguje ok. 

Som si mohol pustit testy...